README.md

```diff
@@ -40,7 +41,7 @@ adhocracy+ is designed to make online participation easy and accessible to every
 
 ### Use postgresql database for testing
 
-run the following command once:
+To use postgresql database in production run the following command once:
```

Comment:
the header for this section indicates is for using postgresql for testing, so should say as the original. some developers prefer to use postgresq instead of sqlite3, me included.

Reply:
Ok I leave that out!